webapp/publisher/snaps/views.py

```diff
@@ -52,6 +53,11 @@
 )
 
 store_api = SnapcraftStoreApi(talisker.requests.get_session(requests.Session))
+launchpad = Launchpad(
```

Comment:
What do you think of the naming - `Launchpad` and `launchpad`, vs `LaunchpadAPI` and `launchpad_api`?

I would normally do the latter, but maybe the former does make sense and is more succinct - `launchpad.get_snaps` is not wrong - maybe `API` is technically redundant, since as we're programming, everything is an application programming interface in one sense.

This would imply, then, that we should be consistent with this naming everywhere and call our SnapStoreAPI simply `SnapStore` etc. (and it should probably be `SnapStore` rather than `Snapcraft`) but this is all way out of scope of this PR. So it's really just me talking to myself.

Please disregard this comment =)

Reply:
Filled https://github.com/canonical-web-and-design/snapcraft.io/issues/2467